src/App.tsx

```diff
@@ -20,20 +26,63 @@ class App extends React.Component<{}, State> {
       './img/9.png',
       './img/10.png',
     ],
+    step: 3,
+    frameSize: 3,
+    itemWidth: 130,
+    animationDuration: 1000,
+    infinite: false,
+  };
+
+  handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
Write methods with correct syntax
```suggestion
  handleChange(event: React.ChangeEvent<HTMLInputElement>) {
```